Sources/Quick/Example.swift

```diff
@@ -71,7 +71,11 @@ final public class Example: NSObject {
         }
         group!.phase = .beforesFinished
 
-        closure()
+        do {
+            try closure()
+        } catch {
+            print("Test \(name) threw unexpected error: \(error.localizedDescription)")
```

Comment:
Once/if #645 is merged this will be trivial, as we could just use `QuickSpec.current.recordFailure`.